core/src/main/java/org/radargun/stages/AbstractDistStage.java

```diff
@@ -26,7 +26,7 @@
  *
  * @author Mircea Markus &lt;Mircea.Markus@jboss.com&gt;
  */
-@Stage(doc = "")
```

Comment:
The documentation for this stage won't be ever seen in schema, therefore it's empty.